### ISSUE TITLE
Schema should not be cached when it changes

### ISSUE THREAD

**Issue by malberts** [ProfessionalWiki/NeoWiki#267]
I was logged in with 2 users. The dev environment's AdminName, and a newly registered user. Both users opened the create Infobox dialog and looked at Population.

then I renamed Population schema's property "Population" to "Populations" using the AdminName user. Then the new user still got the old Schema, leading to an incorrect InfoboxEditor field. The Schema was definitely persisted correctly, because both users see the new name on `Schema:Population`.

[Screencast_20241102_002937.webm](https://github.com/user-attachments/assets/6fa22aea-ad71-4596-9494-4c55ce3e8503)

I'm not sure if this has something to do with our dev environment caching, or if something special happens for the admin user. This could also have been some strange behavior in my private and non-private browser windows. Logging out and in made no difference, but once I closed the browser and logged in again I got the new Schema.

Basically the Schema API requests should not get cached in a way where the browser gets an old/invalid Schema.

I'm logging this so there is a record of a possible cache issue. However, this needs more investigation, so if we're not running into this issue again, then we don't need to spend much time on it.
